### PR TITLE
Fix typos and update WORDLIST

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Review by seabbs — CHANGES_REQUESTED** [review]
LGTM - thanks! Can you add a bullet to the news file on this PR and linking to it (as in the other examples) and then this is good to go.

**Review by adrian-lison — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ This is release is in development. It is not yet ready for production use. If yo
 
 ## Package
 
+- Fixed some typos in `README.md`, `NEWS.md`, the `model.Rmd` vignette and `convolution_matrix()` documentation. The `WORDLIST` used by spelling has also been updated by eliminate false positives. See #221 by @Bisaloo.
```
Comment (seabbs) [inline NEWS.md:7]:
```suggestion
- Fixed some typos in `README.md`, `NEWS.md`, the `model.Rmd` vignette and `convolution_matrix()` documentation. The `WORDLIST` used by spelling has also been updated by eliminate false positives. See #221 by @Bisaloo and reviewed by @seabbs and @adrian-lison.
```

**Review by seabbs — CHANGES_REQUESTED** [review]
Being very fussy on the structure of the news item (we may need more info on this or move responsibility outside of PRs)

**Review by seabbs — APPROVED** [review]
LGTM to me. Thanks for this 😄. I'm happy for it to be merged as and when.

**Comment by Bisaloo** [comment]
An extra commit from `develop` snuck in and I don't like it. It seems like a bug from GitHub and I don't see any way to solve it. Let me try to merge #220 first and rebase and see if it solves it.

**Pull request merged**